third_party/rust/Cargo.toml

```diff
@@ -34,14 +34,14 @@ humantime-serde = "1.1"
 indicatif = "0.16.2"
 is-terminal = "0.4"
 log = "0.4"
-memoffset = "0.6.0"
-mio = { version = "0.7.0", features = ["os-poll", "net", "os-ext"] }
-mio-signals = "0.1.5" # mio-signals 0.2.0 exists, but requires mio 0.8.0
```

Comment:
Do you know why we couldn't upgrade to `0.8.0` when this comment was written?

Reply:
I don't know. This line exists when the file is created. Mio 0.8 removes a bunch of deprecated stuff but we haven't been using the deprecated ones anyway.

Reply:
Fair enough, should be fine.